test/test_torch.py

```diff
@@ -1900,6 +1908,16 @@ def test_torch_from_file(self):
                 t2.fill_(rnum)
                 self.assertEqual(t1, t2, atol=0, rtol=0)
 
+                # release the tensors
+                del s1, t1, s2, t2
+
+            with TemporaryFileName() as fname:
+                assert_with_filename(fname)
+
+            if IS_STDIN_UTF8_ENCODING:
+                with TemporaryDirectoryName(suffix='中文') as dname, TemporaryFileName(dir=dname) as fname:
```

Comment:
Why unicode filename creation should be conditional on the stdin locale environment?

Reply:
This was because the test will fail when the locale encoding is not utf-8. That's why we need to export the `LC_*` variables before running this script.